From: Daniele Tricoli <eriol@debian.org>
Date: Tue, 29 Dec 2020 20:27:27 +0100
Subject: Temporary exclude requests.codes
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

It makes documentation fail using sphinx 3.4.1. I reported the issue
upstream[ยน] but meanwhile we just disable the only class that make
documentation fail.

[ยน] https://github.com/sphinx-doc/sphinx/issues/8616
---
 docs/api.rst | 2 --
 1 file changed, 2 deletions(-)

diff --git a/docs/api.rst b/docs/api.rst
index 93cc4f0..1408c66 100644
--- a/docs/api.rst
+++ b/docs/api.rst
@@ -107,8 +107,6 @@ Cookies
 Status Code Lookup
 ------------------
 
-.. autoclass:: requests.codes
-
 .. automodule:: requests.status_codes
 
 
